_includes/payment-orders-resource.md

```diff
@@ -0,0 +1,277 @@
+## Step 1: Create Payment Order
+
+We start by performing a `POST` request towards the `paymentorder` resource
+with the payer information, such as `consumerProfileRef`.
+Remember to read up on our [URL resource][urls].
+
+{% include alert-risk-indicator.md %}
+
+{% include payment-url.md payment_order=true
+when="selecting the payment instrument Vipps or in the 3-D Secure verification
+for Credit Card Payments" %}
+
+### Request
+
+{% include payment-order-purchase.md %}
```

Comment:
`payment-order-purchase.md` contains mentions of `consumerProfileRef` which is not relevant for Payment Menu.